festim/geometry/geometry.py

```diff
@@ -0,0 +1,23 @@
+from dolfinx import fem
+import ufl
+
+
+class Geometry:
+    """
+    Domain class
+
+    Attributes:
+
+    """
+
+    def __init__(self, mesh=None, subdomains=[]) -> None:
+        """
+        Args:
+
+        """
+        self.mesh = mesh
+        self.subdomains = subdomains
+
+    def define_function_space(self):
+        elements = ufl.FiniteElement("CG", self.mesh.mesh.ufl_cell(), 1)
+        return fem.FunctionSpace(self.mesh.mesh, elements)
```

Comment:
Again, this will only return one type of `FunctionSpace` only.
I don't think this is a method of `Geometry` but rather a method of `HydrogenTransportProblem.`

Which makes me think that maybe we don't need the `Geometry` class at all in the end since it's now only holding two attributes together (subdomains and mesh) and doesn't do anything with it